manifests/1.1.0/opensearch-dashboards-1.1.0.yml

```diff
@@ -7,13 +7,14 @@ components:
   - name: OpenSearch-Dashboards
     repository: https://github.com/opensearch-project/OpenSearch-Dashboards.git
     ref: "1.1"
-  # hybrid repo (workbench)
-  # - name: queryWorkbench
-  #   repository: https://github.com/opensearch-project/sql.git
-  #   ref: main
   - name: alertingDashboards
     repository: https://github.com/opensearch-project/alerting-dashboards-plugin
-    ref: main
+    ref: "1.1"
+  # hybrid repo (workbench)
+  - name: queryWorkbenchDashboards
```

Comment:
Nit: Is this name correct? Do we need dashboards in the name

Reply:
the name here is actually the tmp folder builder.sh will create for the plugin, and it is used for variable PLUGIN_NAME=$(basename $(dirname "$PWD")), and the build process output $PLUGIN_NAME-$VERSION.zip, which needs to match https://github.com/opensearch-project/sql/blob/main/workbench/opensearch_dashboards.json#L2. So in short, it has be this.